packages/graphql_server/src/api/githubApi.ts

```diff
@@ -0,0 +1,94 @@
+import axios, { AxiosResponse } from 'axios'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+type ProjectFounder = {
```

Comment:
Refactor types into a separate file in the types folder named `githubApi.types.ts`